src/genie/libs/parser/nxos/show_track.py

```diff
@@ -0,0 +1,304 @@
+"""show_track.py
+NXOS parser for the following show commands:
+* 'show track'
+"""
+
+import re
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Optional
+
+# =======================
+# Schema for 'show track'
+# =======================
+class ShowTrackSchema(MetaParser):
+    """ Schema for 'show track' """
```

Comment:
Missing `'show track {track_id}'`

Reply:
@LukasMcClelland added the missing line for 'show track {track_id}'

Reply:
@LukasMcClelland made the necessary changes